lib/src/main/java/growthbook/sdk/java/GBContext.java

```diff
@@ -94,13 +98,8 @@ public GBContext(
      * Feature definitions - To be pulled from API / Cache
      */
     @Nullable
-    @Getter(AccessLevel.PACKAGE)
     private JsonObject features;
 
-    private void setFeatures(@Nullable JsonObject features) {
-        this.features = features;
-    }
-
```

Comment:
Could you please clarify the reason of removing setter?

Reply:
it would be taken care by @Data lombok annotation
the current setter is private, we need a public setter which @Data would automatically create.